src/data/food-compounds.ts

```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
       { name: 'O', text: '3' },
     ],
   },
+
+  LMalicAcid: {
+    name: 'L-Malic acid',
+    CASNumber: '97-67-6',
+    id: '5478',
+    formula: 'C4H6O5',
+    SMILES: 'O=C(O)CC(O)C(=O)O',
+    molarWeight: 134.09,
+    criticalTemperature: 849.4,
+    criticalPressure: 7097971.26,
+    criticalVolume: 0.32,
+    criticalCompressibility: 0.32,
+    accentricFactor: 2.22,
+    zRackett: 0.32,
+    prVolumeTransalationCoefficient: 0,
+    srkVolumeTransalationCoefficient: 0,
+    csAccentricFactor: 2.22,
+    csSolubilityParameter: 36.3,
+    csLiquidMolarVolume: 21.6,
+    igEnthalpyOfFormation25C: -6063.38,
+    igGibbsEnergyOfFormation25C: -5130.55,
+    vaporPressureConstantEqnNumber: 0,
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: 28161,
+    idealGasHeatCapacityConstantB: 435.7,
+    idealGasHeatCapacityConstantC: -0.25,
+    idealGasHeatCapacityConstantD: 4.722e-5,
+    idealGasHeatCapacityConstantE: 0,
+    liquidHeatCapacityEqnNumber: 0,
+    liquidHeatCapacityConstantA: 0,
+    liquidHeatCapacityConstantB: 0,
+    liquidHeatCapacityConstantC: 0,
+    liquidHeatCapacityConstantD: 0,
+    liquidHeatCapacityConstantE: 0,
+    liquidHeatCapacityTMin: 0,
+    liquidHeatCapacityTMax: 0,
+    liquidThermalConductivityEqnNumber: 0,
+    liquidThermalConductivityConstantA: 0,
+    liquidThermalConductivityConstantB: 0,
+    liquidThermalConductivityConstantC: 0,
+    liquidThermalConductivityConstantD: 0,
+    liquidThermalConductivityConstantE: 0,
+    liquidThermalConductivityTMin: 0,
+    liquidThermalConductivityTMax: 0,
+    liquidViscocityEqnNumber: 0,
+    liquidViscocityConstantA: 0,
+    liquidViscocityConstantB: 0,
+    liquidViscocityConstantC: 0,
+    liquidViscocityConstantD: 0,
+    liquidViscocityConstantE: 0,
+    liquidDensityEqnNumber: 0,
+    liquidDensityConstantA: 0,
+    liquidDensityConstantB: 0,
+    liquidDensityConstantC: 0,
+    liquidDensityConstantD: 0,
+    liquidDensityConstantE: 0,
+    vaporViscocityEqnNumber: 0,
```

Comment:
In #4 vapor_Viscosity_eqNo is given as "\n ". I don't what should be there , so I am pointing out it